features/mediacontroller.yml

```diff
@@ -0,0 +1,29 @@
+name: MediaController
+description: "The playback of multiple `<video>` or `<audio>` elements can be coordinated by a `MediaController` instance, for example to synchronize a sign-language interpreter track with the main video track."
+spec: https://w3c.github.io/media-source/
```

Comment:
It seems weird to link to Media Source Extensions. The spec has never defined `MediaController`. If you're looking for a link to a spec that still contains `MediaController`, that could perhaps be the [supserseded HTML5 spec](https://www.w3.org/TR/2018/SPSD-html5-20180327/embedded-content-0.html#synchronising-multiple-media-elements)

Reply:
Oh that was a mistake on my part. I didn't intend to link to this spec. Thanks for the link. I will replace.